frontend/src/components/common/button/styles.module.scss

```diff
@@ -15,6 +20,11 @@
   cursor: pointer;
 }
 
+.iconButton {
+  min-width: 15px;
+  min-height: 15px;
+}
```

Comment:
Not sure if we need this style for label

Reply:
Otherwise icons get as big as buttons and it breaks the view